wetmap/src/components/googleMap.jsx

```diff
@@ -63,6 +61,9 @@ import DiveSite from './newModals/diveSite/index';
 // import ShopModal from './modals/shopModal';
 import ShopModal from './newModals/shopModal/index';
 
+
+import { getSiteNamesThatFit } from '../supabaseCalls/diveSiteSupabaseCalls';
```

Comment:
Question: is this a call that retrieves shortened forms of names stored in the DB? Or what exactly is this import

Reply:
It's a simple query that takes whatever the user has typed into the search field and returns any dive sites in the Db that have names that fits that passed in string. It returns the entire record for each though not a shortened set of data, mostly because some other info about the dive sites is needed to then move the map to the dive site that is chosen from the list 
Shorter answer: its a data based query call, yes :)

Reply:
oh yes i didn't notice that I had let that in there, ill push a change to get rid of it and a few other unused imports while I'm at it